go.work

```diff
@@ -9,4 +9,4 @@ use (
 	./router-tests
 )
 
-//replace github.com/wundergraph/graphql-go-tools/v2 => ../../graphql-go-tools/v2
+//replace github.com/wundergraph/graphql-go-tools/v2 => ../graphql-go-tools/v2
```

Comment:
please don't change this file
ideally, we could add it to the gitignore, but have a default somehow

Reply:
I wouldn't normally commit this, but based on the default replace in router go.mod, the extra `../` seems wrong.

I.e., router is one dir deeper and the `go.mod` is `../../`, but here we're one dir up and it's the same relative path.